scrum.py

```diff
@@ -132,12 +166,12 @@ def get_all_board_members(self):
         return json.loads(response.text)
 
 
-    def get_all_board_labels(self):
+    def get_all_board_labels(self, config):
         url = "https://api.trello.com/1/boards/627c210aa0ed4a48c3dd069c/labels"
```

Comment:
Do not forget to replace this hardcoded 627c210aa0ed4a48c3dd069c id

Reply:
fixed